src/components/OnboardingModal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <ModalDefault
+    class="about-dex-modal"
+    close
+    @close="resolve"
+  >
+    <div class="wrapper">
+      <h1 class="info-headline">
+        {{ $t('onboardingModal.headline') }}
+      </h1>
+      <p class="info-text">
+        {{ $t('onboardingModal.body') }}
+      </p>
+      <p class="info-text">
+        <a
+          href="https://dex.superhero.com"
+          target="_blank"
+        >https://dex.superhero.com</a>
```

Comment:
Shall we use aepp.dex.superhero.com instead?

Reply:
the url is not final at the moment but will be basically leading to an FAQ page

Reply:
final url is in now